### ISSUE TITLE
Put together a Fast Transformer model

### ISSUE THREAD

**Issue by Rishit-dagli** [Rishit-dagli/Fast-Transformer#5]
- [x] Implement Positional embeddings
- [x] Create FastAttention (#4) and FeedForward (#3 ) layers
- [x] Weight tie projections
- [x] Converts to logits
